src/database/DataTypes/Transaction.js

```diff
@@ -419,8 +419,14 @@ export class Transaction extends Realm.Object {
         expiryDate,
         costPrice,
         sellPrice,
+        doses,
+        location,
+        vaccineVialMonitorStatus,
       } = transactionBatch;
 
+      itemBatch.applyVvmStatus(database, vaccineVialMonitorStatus);
+      itemBatch.applyLocation(database, location);
+
```

Comment:
same as the stocktake, just setting the underlying item batches location adn vvm status when finalising a transaction